src/app/desktop/windows/file-manager/file-manager.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit, Type } from '@angular/core';
+import { WindowComponent, WindowDelegate } from '../../window/window-delegate';
+import { WebsocketService } from '../../../websocket.service';
+
+@Component({
+  selector: 'app-file-manager',
+  templateUrl: './file-manager.component.html',
+  styleUrls: ['./file-manager.component.scss']
+})
+export class FileManagerComponent extends WindowComponent implements OnInit {
+  public files: string[];
+
+  constructor(private websocket: WebsocketService) {
+    super();
+  }
+
+  ngOnInit() {
+    this.update();
+
+    // update shown files every 1 sec
+    setInterval(() => this.update(), 1000);
```

Comment:
So what?! You shouldnt send the same request every second.
Change this!

Reply:
Maybe we should get a notification when a file is changed/created